app/models/prison.rb

```diff
@@ -15,6 +17,18 @@ def self.find_by_id(id)
     PrisonVisits::Api.instance.get_prison(id)
   end
 
+  def closed?
+    closed
+  end
+
+  def enabled?
+    !closed? && !private?
```

Comment:
If we are going to deploy this PR first then `closed` and `private` won't be returned by the API yet.

A solution could be to reject not 'enabled` field that comes from the prisons API temporarily, then this can be deployed first, followed by pvb2 and finally remove the reject part.